src/tests/token/erc1155/test_dual1155.cairo

```diff
@@ -298,22 +286,22 @@ fn test_dual_supports_interface_exists_and_panics() {
 //
 
 #[test]
-#[ignore]
+#[ignore] // REASON: foundry entrypoint_not_found error message inconsistent with mainnet.
 fn test_dual_balanceOf() {
```

Comment:
Is this the reason though? Assuming that this error is consistent with mainnet, this would still need to be ignored because the test needs try/catch support, right?

Reply:
Not really, safe dispatchers work in foundry, meaning that `call_contract_syscall` returns the actual result, and we can check for the error in the `try_selector_with_fallback`. This functions will work if we update the error in this method to the real error message instead of ENTRYPOINT_NOT_FOUND, but we can't update the the foundry version, because is not the one used in mainnet.

Reply:
Ahhh thank you, I see what you mean now